dropwizard-logging/src/main/java/io/dropwizard/logging/FileAppenderFactory.java

```diff
@@ -199,7 +199,6 @@ public boolean isMaxFileSizeSettingSpecified() {
         appender.setPrudent(false);
         appender.addFilter(levelFilterFactory.build(threshold));
         getFilterFactories().stream().forEach(f -> appender.addFilter(f.build()));
-        appender.stop();
```

Comment:
If this line was not removed, test `DefaultLoggingFactoryTest::testConfigure` will fail